<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_set_ibp</class>
 <widget class="QDialog" name="Dlg_set_ibp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>有创血压设置</string>
  </property>
  <widget class="QComboBox" name="m_cmbb_biaochi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_moshi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>115</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_chnl1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>8</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_spb_chnl2_high">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波形速度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通  道  1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_p1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>195</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道2低标尺</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波形标尺</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_speed">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道1高标尺</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_p2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>45</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通  道  2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>滤波模式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_unit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>185</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_spb_chnl1_low">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>115</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ibp_jiaoling">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>有创血压校零</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>125</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道1低标尺</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>55</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>P2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道2高标尺</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>单        位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>P1</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_spb_chnl2_low">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>185</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_chnl2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>45</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>8</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_spb_chnl1_high">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
